examples/zone-printer/README.md

```diff
@@ -1,258 +1,252 @@
-terminal:
+    After your project is created, run the following commands in your
+    terminal:
 
-```shell
-PROJECT=<your-project-name>
-```
+    ```shell
+    PROJECT=<your-project-name>
+    ```
 
-### 0.2 `gcloud` installed and initialized
+2. **Set up `gcloud`**:
 
-You can install `gcloud` by following the instructions at
-[Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
-initialize it by running the following commands:
+    You can install `gcloud` by following the instructions at
+    [Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
+    initialize it by running the following commands:
 
-```shell
-gcloud init # Set $PROJECT as the default project
-gcloud auth login
-gcloud auth application-default login
-```
+    ```shell
+    gcloud init # Set $PROJECT as the default project
+    gcloud auth login
+    gcloud auth application-default login
+    ```
 
-### 0.3 kubectl
+3. Set up **`kubectl`**:
 
-You can install `kubectl` by running:
+    You can install `kubectl` by running:
 
-```shell
-gcloud components install kubectl
-```
+    ```shell
+    gcloud components install kubectl
+    ```
 
-### 0.4 Kubernetes clusters on GCP
+## 1. Create Kubernetes Clusters
 
-#### Existing clusters
+`kubemci` requires Kubernetes clusters that are v1.8.1 or newer. (You can check
+your cluster version using the `kubectl version` command.)
 
-> **Note:**: If you don't already have Kubernetes cluster on GCP,
-skip to the next subsection on [creating new clusters](#creating-new-clusters).
+You will need at least two Kubernetes clusters in two different compute [zones]
+to verify that `kubemci` works.
 
-`kubemci` requires Kubernetes clusters that are v1.8.1 or newer. Please
-ensure that all the clusters you are using in this tutorial satisfy the
-minimum version requirement. You can check your cluster version using
-the `kubectl version` command.
+[zones]: https://cloud.google.com/compute/docs/regions-zones/
 
-If you already have clusters with their credentials in your local
-kubeconfig (`$HOME/.kube/config` by default) file, you need to ensure
-that you either only have clusters you are going to use for this
-tutorial in your default kubeconfig file or create a new kubeconfig
-file that just contains credentials for those clusters. Assuming that
-the clusters you are going to use for this tutorial are called
-`us-east`, `eu-west` and `asia-east`.
+#### Using existing GKE clusters
 
-List the contexts available (note the names of the ones you wish to
-use with kubemci)
+> **Note:** If you don't already have Kubernetes cluster on GCP,
+skip to the next subsection on [creating new clusters](#creating-new-gke-clusters).
 
-```shell
-$ kubectl config get-contexts -o name
-us-east
-eu-west
-asia-east
-i-dont-care
-doesnt-exist
-```
+If you already have existing Kubernetes Engine clusters, run `get-credentials`
+with them to create a `clusters.yaml` file with the connection information of
```

Comment:
maybe my exposure has been limited, but I've never seen the KUBECONFIG file with a ".yaml" extension. Is that something people do? Otherwise, I would drop the extension.

Reply:
I think it has no significance. It just informs the users that it's a YAML file so if they ever wonder what's in it, they now guess it's a YAML file.